doc/advanced_configuration.rst

```diff
@@ -1,42 +1,70 @@
+=============
+Configuration
+=============
+
+List of config options
 ======================
-Advanced Configuration
-======================
 
-Here are the personal configurations that you can modify within Sphinx-Gallery.
+Most sphinx-gallery configuration options are set in the Sphinx ``conf.py``
+file:
+
+- ``examples_dirs`` and ``gallery_dirs`` (:ref:`multiple_galleries_config`)
+- ``filename_pattern`` (:ref:`build_pattern`)
+- ``subsection_order`` (:ref:`sub_gallery_order`)
+- ``reference_url`` (:ref:`link_to_documentation`)
+- ``backreferences_dir`` and ``doc_module`` (:ref:`references_to_examples`)
+- ``default_thumb_file`` (:ref:`custom_default_thumb`)
+- ``line_numbers`` (:ref:`adding_line_numbers`)
+- ``download_section_examples`` (:ref:`disable_joint_download`)
+- ``plot_gallery`` (:ref:`without_execution`)
+- ``abort_on_example_error`` (:ref:`abort_on_first`)
+- ``expected_failing_examples`` (:ref:`dont_fail_exit`)
+
+Some options can also be set or overridden on a file-by-file basis:
+
+- ``# sphinx_gallery_line_numbers`` (:ref:`adding_line_numbers`)
+- ``# sphinx_gallery_thumbnail_number`` (:ref:`choosing_thumbnail`)
+
+Some options can be set during the build execution step, e.g. using a Makefile:
+
+- ``make html-noplot`` (:ref:`without_execution`)
+- ``make html_abort_on_example_error`` (:ref:`abort_on_first`)
+
+And some things can be tweaked directly in CSS:
+
+- ``.sphx-glr-thumbcontainer`` (:ref:`setting_thumbnail_size`)
```

Comment:
all our CSS is in sphinx_gallery/_static/gallery.css, and every user can always overload that with their own CSS. I left PR #210 forgotten, at some point we shall go back to it.